src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -12,14 +12,6 @@
   <PropertyGroup Condition="'$(MSAL_DESKTOP_ONLY_DEV)' == ''">
     
     <TargetFrameworkNetStandard>netstandard1.3</TargetFrameworkNetStandard>
-    <TargetFrameworkNetDesktop45>net45</TargetFrameworkNetDesktop45>
```

Comment:
Please undo this, we cannot remove legacy frameworks, except maybe net45. I'd prefer to remove net45 in a separate commit.